mmdet/datasets/custom.py

```diff
@@ -96,6 +99,13 @@ def __init__(self,
         self.mask_transform = MaskTransform()
         self.numpy2tensor = Numpy2Tensor()
 
+        # if use extra augmentation
+        if extra_aug is not None:
+            self.extra_aug = ExtraAugmentation(
+                img_norm_cfg.mean, img_norm_cfg.to_rgb, **extra_aug)
```

Comment:
Set `mean`, `std` and `to_rgb` in config files so that we can initialize it with `ExtraAugmentation(**extra_aug)`